ios/RNSScreen.m

```diff
@@ -6,6 +6,10 @@
 #import "RNSScreenStackHeaderConfig.h"
 #import "RNSScreenWindowTraits.h"
 
+// Imported only to make project build before RNSScreenView is merged
+// with RNSScreenComponentView
+#import "RNSScreenController.h"
```

Comment:
Won't you need to import it anyway if we want ot have the controller in separate file?